WordPress/Classes/ViewRelated/Post/PostSharingController.swift

```diff
@@ -145,3 +147,51 @@ import SVProgressHUD
 
     typealias PopoverAnchor = UIPopoverPresentationController.PopoverAnchor
 }
+
+private class CopyLinkActivity: UIActivity {
+    var activityItems = [Any]()
+    private var url = URL(string: "")
+
+    override var activityTitle: String? {
+        return NSLocalizedString(
+            "share.sheet.copy.link.title",
+            value: "Copy Link",
+            comment: "Title for the \"Copy Link\" action in Share Sheet."
+        )
+    }
+
+    override var activityImage: UIImage? {
+        return UIImage(systemName: "doc.on.doc")
+    }
+
+    override var activityType: UIActivity.ActivityType? {
+        return UIActivity.ActivityType(rawValue: "copy.link.activity")
```

Comment:
```suggestion
        return UIActivity.ActivityType.copyToPasteboard
```

Reply:
This is different from `copyToPasteboard` action isn't it? After all we show both activities.

Reply:
I might be missing something. Is "copy.link.activity" a custom string that we chose?

Reply:
@alpavanoglu I might be missing something. Is "copy.link.activity" a custom string that we chose?

Reply:
The API is a bit odd. `ActivityType` is not an enum but a struct. From what I understood, we either pick one of the predefined actions or provide a custom string for the custom activities. That's what I understood at least. Lmk if that's not correct and I can change it

Reply:
Okay got it 👍